Makefile

```diff
@@ -21,8 +21,19 @@ static:
 #	CGO_ENABLED=0 GOOS=linux go build -ldflags "-s" -a -installsuffix cgo -o $(GOBIN)/api-gateway-config-supervisor-static ./
 	CGO_ENABLED=0 go build -ldflags "-s" -a -installsuffix cgo -o $(GOBIN)/api-gateway-config-supervisor-static ./
 
-docker:
-	docker build -t adobeapiplatform/api-gateway-config-supervisor .
+DOCKER_TEMP := $(shell mktemp -u ./.Dockerfile.XXXXXXXXXX)
```

Comment:
The intent here is to provide a `debug` configuration option when building the docker container ?